src/ripple/app/main/GRPCServer.cpp

```diff
@@ -0,0 +1,408 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
```

Comment:
The copyright on this and other new files is a little dated for new files.

Reply:
Should I just change the year to 2020?